apps/docs/tests/support/pageObject/DetailsPageObject.ts

```diff
@@ -0,0 +1,12 @@
+export class DetailsPageObject {
+  get frameworkName() {
+    return cy.get('main').find('div[class="mb-8"]').first().find('span').first();
+  }
+  header(name: string) {
```

Comment:
```suggestion
  get frameworkName() {
    return cy.get('main').find('div[class="mb-8"]').first().find('span').first();
  }
  
  header(name: string) {
```
We probably should introduce eslint rules here